brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Remove default styling from various blocks.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/iHKt2gD8SSQ/d117346a-4e6e-4f61-b78a-a83edb66d0da?view=0a6f113f-f5c3-4b4e-82f0-1a6fc4831d3e&t=attributes

### Types of changes




- Imrovement

### How has this been tested?

-  Case 1 - Backward compatibility
- Drag and drop info-box, Select CTAType as a button save and refresh the page.
- Switch to `default-block-design-as-per-theme` branch and check everything works fine.
- Drag and drop FAQ, TOC and Post grid, Masonry by backward compatibility.
- Case 2 - Set Colors and Typography from Theme 
- Drag and drop blocks and check Is theme colors and typography is applying fine in the editor as well as frontend.
- Case 3 - By Changing different Theme other than Astra.

### Checklist:
